pb/transaction.go

```diff
@@ -135,3 +136,24 @@ func (m *SigChainTxn) ToMap() map[string]interface{} {
 		"Submitter": common.BytesToUint160(m.Submitter),
 	}
 }
+
+type SubTxnDBChangePoint uint32
+
+const (
+	LimitMetaCP SubTxnDBChangePoint = 124000
```

Comment:
How about move this const SubTxnDBChangePoint to config.go?

Reply:
Done.